API.yaml

```diff
@@ -1278,6 +1290,8 @@ paths:
                   requested_table_index: 0
                   requested_table_count: 100
                   requested_table_column_ids: [0, 1, 2]
+                  requested_search_expression: { "1": "cpu.*"}
```

Comment:
I confirmed that this works also (as a string key representing a parsable number) in the trace-server.

Just because I have another comment already: in the real JSON there is no space before the "1"... ;)

Reply:
Done